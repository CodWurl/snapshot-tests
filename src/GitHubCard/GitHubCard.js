import Card from "react-bootstrap/Card"
import Card from "react-bootstrap/Card"


function GitHubCard() {
    return (
        <div className="card">
          <Card style={{ width: "18rem"}}>
          {/* Use your GitHub info instead, but Mickey Mouse is used as an example */}
          <Card.Img variant="top" src={MickeyMouse} />
          {/* Source: https://en.wikipedia.org/wiki/Mickey_Mouse */}
          <Card.Body>
              <Card.Title>Code World</Card.Title>
              <Card.Text>
              I am an aspiring Software Engineer/Developer
              </Card.Text>
          </Card.Body>
          </Card>
        </div>
    );
  }
  
  export default GitHubCard