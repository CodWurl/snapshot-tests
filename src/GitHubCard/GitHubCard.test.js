import GitHubCard from './src/GitHubCard/GitHubCard.js'
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const tree = renderer.create(<GitHubCard />).toJSON()
  expect(tree).toMatchSnapshot()
})