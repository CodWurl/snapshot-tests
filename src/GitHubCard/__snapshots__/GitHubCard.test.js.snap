// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-middle"
      src="Final_Fantasy.gif"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Code World
      </div>
      <p
        className="card-text"
      >
        I am an aspiring Software Engineer/Developer
      </p>
    </div>
  </div>
</div>
`;
